### ISSUE TITLE
Note somewhere in the doc to clear up the confusion about "context"

### ISSUE THREAD

**Issue by Huy-Ngo** [McSinyx/palace#125]
In palace, there is a class named `Context`. To my understanding, `Context` means the environment where the sound is played. This should not be confused with [with statement context manager](https://docs.python.org/3/reference/datamodel.html#context-managers) - to be honest, I was confused when I was resolving issues related to this class too. I'm not sure if this would be clear to users.

Should we maybe add a glossary explaining the name of classes and their properties?

**Comment by MostDeadDeveloper** [comment]
I would not mind working on this will that be fine?

**Comment by Huy-Ngo** [comment]
Of course, though I'm not sure which is a better way to deal with it - including it in each class's docstring or writing a separate doc section for that.

**Comment by MostDeadDeveloper** [comment]
Would you mind giving me a list of specific classes that you would like to be added to this Glossary?

**Comment by MostDeadDeveloper** [comment]
I will work on this, this week, just gonna have to deal some other stuff

**Comment by McSinyx** [comment]
Ooh this is another thing I thought I commented but didn't.

> Context means the environment where the sound is played. This should not be confused with with statement context manager

Agree

> Should we maybe add a glossary explaining the name of classes and their properties?

I'm -0 on this depending on how good the execution's gonna be.  Currently we already have definitions overlapping in reference and tutorial, and there's also an index for the list of, ehm, [everything](https://mcsinyx.github.io/palace/genindex.html).  I feel like a note in [Context](https://mcsinyx.github.io/palace/reference/context.html#palace.Context) clarifying what a might be good enough.  However that's to be phrased, let's avoid putting URLs in docstrings (I'm not even sure if napoleon supports that to begin with).

@MostDeadDeveloper, please try looking into GH-46 or GH-93 instead, whose requirement are more obvious (and that you can learn more about the module's functionality, if you want to hang around after the fest), since this issue is to be discussed further.
